data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,59 @@ mission "Remnant: Shattered Light 6"
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
+	landing
+	source
+		government "Remnant"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	npc "scan outfits"
+		government "Korath"
+		personality waiting target
+		system
+			system "Kor Zena'i"
+		ship "Tubfalet" "Ra'puk At 295"
+		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
+	npc
+		government "Korath"
+		personality waiting
+		system
+			system "Kor Zena'i"
+		fleet
+			names "korath"
+			cargo 3
+			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
+			outfitters "Korath Basics" "Korath Exiles"
+			variant
+				"'olofez" 3
+	on offer
+		event "remnant: tubfalet appearance" 20 31
+		conversation
+			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
+			choice
+				`	"Yes, I could do that."`
+				`	"I would rather not."`
+					decline
+			`	He signs appreciatively. "We appreciate your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`
+				accept
+	on enter "Kor Zena'i"
+		dialog
+			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This would be the ship that Chilia would like a scan of.`
+	on complete
+		payment 150000
+		conversation
+			`Chilia is waiting for you when you land. He looks over the scan results, nodding thoughtfully, and listens to your account of visiting Kor Zena'i.`
```

Comment:
Why do I need to bring the scan back to him at all?
In that other mission, someone said something about a `Quantum Entangled Network`. Why can't the scan data just be transmitted along that? He should've seen this information a week ago.
Maybe he wants my thoughts on it or otherwise wants to discuss it in person, but the data could at least be accessed sooner.

Reply:
This section has been updated to make it clear that Chilia already has the data, and is interested in what *you* have to say about the ship. Thanks for bringing this up!